package.json

```diff
@@ -12,7 +12,7 @@
   },
   "private": true,
   "dependencies": {
-    "@geneontology/ribbon": "1.5.3",
+    "@geneontology/ribbon": "^1.5.7",
```

Comment:
I feel a little uncomfortable about the compatible caret here. No disrespect, but this package has had a history of releasing backwards-incompatible changes as patch versions. I'd rather be more deliberate about using specific versions.

Reply:
I agree.  I think that's what we had by default as well.   @paaatrick  Do you want to test this against the expression ribbon?